ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/LoadBalancerContext.java

```
@@ -245,12 +245,18 @@ private ClientException generateTimeoutNIWSException(String uri, Throwable e){
     }
 
     private void recordStats(ServerStats stats, long responseTime) {
+    	if (stats == null) {
+    		return;
+    	}
         stats.decrementActiveRequestsCount();
```

😆 were we also swallowing null pointer exceptions?